operator/cmd/mesh/operator-common.go

```diff
@@ -55,7 +55,7 @@ const (
 // isControllerInstalled reports whether an operator deployment exists in the given namespace.
 func isControllerInstalled(cs kubernetes.Interface, operatorNamespace string, revision string) (bool, error) {
 	orn := operatorResourceName
-	if revision != "" {
+	if revision != "" && revision != "default" {
```

Comment:
Can you do all these things in Args? I don't think this is the right layer to do the arg validation